pkgs/top-level/all-packages.nix

```diff
@@ -21661,12 +21661,9 @@ in
 
   minecraft = callPackage ../games/minecraft { };
 
-  minecraft-server = minecraft-server_1_14;
+  minecraft-server = minecraft-server-versions.latest;
 
-  inherit (callPackages ../games/minecraft-server { })
-    minecraft-server_1_14
-    minecraft-server_1_13_2
-    minecraft-server_1_12_2;
```

Comment:
Would you mind adding an alias for these? I think it's good practice to strive for backwards compatibility, deprecation and better error messages

Reply:
I would mind:

- It's cruft
- We have precedent for removing old minecraft server versions without notice: c7b36b71efbe541ed8517536de2ffec3db3d9277
- This pull request is to master, which goes to unstable, and isn't expected to maintain backwards compatibility.

Reply:
Okay well I'm okay with doing it in this case, because these versions have only been introduced recently and aren't in the last release.